pkg/cli/admin/mustgather/mustgather.go

```diff
@@ -95,6 +98,7 @@ func NewMustGatherCommand(f kcmdutil.Factory, streams genericiooptions.IOStreams
 	cmd.Flags().BoolVar(&o.HostNetwork, "host-network", o.HostNetwork, "Run must-gather pods as hostNetwork: true - relevant if a specific command and image needs to capture host-level data")
 	cmd.Flags().StringSliceVar(&o.Images, "image", o.Images, "Specify a must-gather plugin image to run. If not specified, OpenShift's default must-gather image will be used.")
 	cmd.Flags().StringSliceVar(&o.ImageStreams, "image-stream", o.ImageStreams, "Specify an image stream (namespace/name:tag) containing a must-gather plugin image to run.")
+	cmd.Flags().StringVar(&o.GatherArgs, "args", "", "Extra arguments to pass to the must-gather script.")
```

Comment:
As far as I know, there wasn't any request to pass arguments to the must-gather images(including the default one) and it seems like a requirement very specific to a custom must-gather image. 

To preserve simplicity, I'd not prefer adding a new flag in must-gather command, instead I'd recommend using environment variables to manage.

Reply:
The arguments are also only passed to the 'gather' script. 
Why not just use `oc adm must-gather --image IMAGE -- [gather|OTHER_SCRIPT] ARGUMENT` to run the command?

Reply:
Ack, As this functionality is only supported my odf-must-gather, we will advise customers to use the custom command syntax as of now. The only redundancy I see going this way is providing the abs path to the gather script before the args. `-- /usr/bin/gather --args`.

Regards